sdk/identity/azure-identity/samples/user_authentication.py

```diff
@@ -32,11 +34,13 @@
 client = SecretClient(VAULT_URL, credential)
 secret_names = [s.name for s in client.list_properties_of_secrets()]
 
-# With persistent caching enabled, an authentication record stored by your application enables
-# credentials to access data from past authentications. If the cache contains sufficient data,
-# this eliminates the need for your application to prompt for authentication every time it runs.
+# An authentication record stored by your application enables other credentials to access data from
+# past authentications. If the cache contains sufficient data, this eliminates the need for your
+# application to prompt for authentication every time it runs.
```

Comment:
I would say that maybe we should mention that allowing other credentials to access this data is optional, except that's probably outside the scope of a short sample. Also, that information is covered in the code snippet for `TokenCachePersistenceOptions`, so I think that should be sufficient